metadata-ingestion/src/datahub/ingestion/sink/datahub_rest.py

```diff
@@ -346,6 +347,11 @@ def emit_async(
             RecordEnvelope(item, metadata={}), NoopWriteCallback()
         )
 
+    def flush(self) -> None:
```

Comment:
I would add some message that we are flushing
And, from time to time, pending requests